onnx/backend/test/case/node/averagepool.py

```diff
@@ -455,3 +455,34 @@ def export_averagepool_2d_ceil() -> None:
         y = np.array([[[[6, 7.5], [12, 13.5]]]]).astype(np.float32)
 
         expect(node, inputs=[x], outputs=[y], name="test_averagepool_2d_ceil")
+
+    @staticmethod
+    def export_averagepool_2d_dilations() -> None:
+        """
+        input_shape: [1, 1, 4, 4]
+        output_shape: [1, 1, 2, 2]
+        """
+        node = onnx.helper.make_node(
+            "AveragePool",
+            inputs=["x"],
+            outputs=["y"],
+            kernel_shape=[3, 3],
+            strides=[1, 1],
+            dilations=[2, 2],
+            ceil_mode=True,
+        )
+        x = np.array(
```

Comment:
I don't know if it is better to let it that way or to replace it by `(np.arange(16) + 1).astype(np.int64).reshape((1, 1, 4, 4))`.

Reply:
Good point, done.